src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -161,8 +161,6 @@ public void start(final Map<String, String> parsedConfig) {
 
     this.DYNAMIC_LOGGER.setLoggerInstanceTag(this.getTaskLoggingTag());
 
-    this.DYNAMIC_LOGGER.info("starting task...");
```

Comment:
Its not strictly necessary since kafka also emits a start task log. I'll add it back in though, since it would be useful for determining if an issue is in kafka's code or ours. 

[2023-01-13 00:28:50,275] INFO [Worker clientId=connect-1, groupId=compose-connect-group] Starting task SnowflakeSinkConnector_rcheng-1 (org.apache.kafka.connect.runtime.distributed.DistributedHerder)
[2023-01-13 00:28:50,275] INFO [Worker clientId=connect-1, groupId=compose-connect-group] Starting task SnowflakeSinkConnector_rcheng-0 (org.apache.kafka.connect.runtime.distributed.DistributedHerder)